### PR TITLE
Example workspace enhancements

### PR DESCRIPTION
**Pull request by jirimosinger**

### CODE REVIEW

File: example_workspace/crazy.hlasm [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
-    MAC
-    
+         MAC
 
-    AGO .SKIPRUBBISH
-    Anything in here is just skipped.
-    And it is still valid hlasm source code.
-.SKIPRUBBISH ANOP
+**************************  RUBBISH SKIPPING  *************************
```
Comment (slavek-kucera) [inline example_workspace/crazy.hlasm:11]: At this point we can call it a block comment, right?

File: example_workspace/crazy.hlasm [hunk 2]
```diff
@@ -1,15 +1,15 @@
-* MACRO CYCLES
-* it is possible to recursively call macro from itself
-* the number of calls is limited in HLASM 
-    MACRO
-    MAC
-    MAC
-    MEND 
+***************************  INFINIT CYCLES  **************************
```
Comment (slavek-kucera) [inline example_workspace/crazy.hlasm:1]:
```suggestion
***************************  INFINITE CYCLES  **************************
```

File: example_workspace/factorial.hlasm [hunk 3]
```diff
@@ -1,35 +1,55 @@
...
+* Hence, the calculation of 8! takes place instead:
+.NEXT    AIF   ('&SYSPARM' NE 'EIGHT').FINAL
+         FACT  8
+         AGO   .FINAL
+
+* Finally, we can print the result in the 'PROBLEMS' tab by using MNOTE
+* instruction. Try using different MNOTE levels!
+.FINAL   MNOTE 2,'Result is &RES'
+
+         END
```
Comment (slavek-kucera) [inline example_workspace/factorial.hlasm:55]: I suggest putting the newlines at the end.

File: example_workspace/crazy.hlasm (hunk 1 above)
Comment (jirimosinger) [reply example_workspace/crazy.hlasm]: Agreed. I've merged this with comments.hlasm and renamed crazy.hlasm to just infinite.hlasm

**Review by slavek-kucera — APPROVED** [review]

**Pull request merged**